api/python/stat_all.md

```diff
@@ -36,9 +36,7 @@ with the following fields
 - `unit`: the [`unit`](https://datacommons.org/browser/unit) of the observations, if it exists.
 - `scalingFactor`: the [`scalingFactor`](https://datacommons.org/browser/scalingFactor) of the observations, if it exists.
 
-**Raises**
-
-- `ValueError` - If no statistical value found for any `Place` and `StatisticalVariable` combinations.
+If no statistical value can be found for any `Place` and `StatisticalVariable` combinations, `NaN` is returned.
```

Comment:
If you run dc.get_stat_all(['badGeoId'], ['BadStaVar'])

the time series is an empty dict. I'm not really sure how to English this, but basically when there is no data for a particular Place and StatVar time series, the time series dict is empty.